### PR TITLE
Make the title of graphs more human-readable

### PR DESCRIPTION
**Pull request by YinengWang**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -131,9 +131,9 @@ def train(model, criterion, optimizer, n_epochs, train_loader, test_loader=None,
             correct_wrong_per_epoch.append(correct_in_wrong)
             memorized_wrong_per_epoch.append(memorized_in_wrong)
             incorrect_wrong_per_epoch.append(incorrect_in_wrong)
-            wandb.log({"correct_clean": correct_in_clean, "incorrect_clean": incorrect_in_clean,
-                       "correct_wrong": correct_in_wrong, "memorized_wrong": memorized_in_wrong,
-                       "incorrect_wrong": incorrect_in_wrong}, step=batch_ct)
+            wandb.log({"correct_in_clean_labels": correct_in_clean, "incorrect_in_clean_labels": incorrect_in_clean,
+                       "correct_in_wrong_labels": correct_in_wrong, "memorized_in_wrong_labels": memorized_in_wrong,
+                       "incorrect_in_wrong_labels": incorrect_in_wrong}, step=batch_ct)
```
Comment (rlindsberg) [inline main.py:136]: So "correct in clean labels" etc. means:
Percentage of correctly predicted labels among all clean labels etc.?

Comment (YinengWang) [reply main.py:136]: Yes! Do we need to make it more clear?

**Review by rlindsberg — APPROVED** [review]
> Yes! Do we need to make it more clear?

We can leave it for now and see if the plots look good in the report.

**Comment by rlindsberg** [comment]
@Alfred-N

**Review by Alfred-N — APPROVED** [review]
Yes, this is understandable, I approve
